packages/react-core/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@aws-amplify/ui": "3.14.0",
-    "@xstate/react": "3.0.0"
+    "@xstate/react": "3.0.1"
```

Comment:
Just curious, is there a reason for this bump?

Reply:
The latest version of `@xstate/react` has a peer dependency on the same minimum version of `xstate` that we use. This is needed for proper deduping to happen and for `@xstate/react` to not use a different instance of the package than `@aws-amplify/ui` does.